src/Api/Controllers/AuthController.php

```diff
@@ -357,7 +357,7 @@ public function recover(): Response
     public function reset(string $key) : Response
     {
         //is the key empty or does it existe?
-        if (empty($key) || !$userData = Users::findFirst(['user_activation_forgot = :key:', 'bind' => ['key' => $key]])) {
+        if (empty($key) || !$userData = Users::findFirst(['user_activation_forgot = ?0', 'bind' => ['key' => $key]])) {
```

Comment:
@rwhite27 if we change the :key: to ?0 , we need to change the bind to [$key] , remove the 'key' index